Description: Fix Yacc build
 This patch adjusts a Makefile.am to correctly build Yacc files (fixes build
 error after reautoconfiguration)
Author: Roland Stigge <stigge@antcom.de>

--- alliance-5.1.1.orig/exp/src/Makefile.am
+++ alliance-5.1.1/exp/src/Makefile.am
@@ -2,7 +2,8 @@
 
 bin_PROGRAMS = exp 
 
-YACC = @YACC@ -d -v --debug
+BUILT_SOURCES = expy.h
+AM_YFLAGS = -d -v --debug
 CFLAGS = @CFLAGS@ @ALLIANCE_CFLAGS@
 exp_LDADD =
 
